k8s/helm/values.yaml

```diff
@@ -10,6 +10,9 @@ image:
   pullPolicy: Always
 
 submissionBucket: ""
+gtmId: ""
+gtmAuth: ""
+gtmPreview: ""
```

Comment:
Expanding `gtm` to `googleTagManager` would be clearer.

Reply:
abbreviations are bad

Reply:
Done